### PR TITLE
docs: identify language in some markdown blocks

### PR DESCRIPTION
**Pull request by erights**
I really like syntax highlighting.

See https://github.com/endojs/endo/pull/1800

Reviewers, I suggest reviewing with "hide whitespace". The occasional invisible whitespace normalizations were due to my editor, not me.

### CODE REVIEW

File: packages/SwingSet/docs/delivery.md [hunk 1]
```diff
@@ -1281,9 +1281,9 @@ Vat-2 then gets a `dispatch.deliver(target=o+3001, msg={method: "foo", args:
 * `syscall.resolve(to=Fulfill())`, showing how queued messages are then rejected
 * `syscall.resolve(to=Rejection())`, ditto
 
-```
+```js
 p1 = make_promise();
-x~.foo(p1);
+E(x).foo(p1);
```
Comment (erights) [inline packages/SwingSet/docs/delivery.md:1286]: Unrelated drive-by change

File: packages/ERTP/docs/INPUT_VALIDATION.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Object.defineProperty(object1, 'value', {
  },
 });
 ```
-```sh
+```console
```
Comment (erights) [inline packages/ERTP/docs/INPUT_VALIDATION.md:79]: Not technically correct, but it still produces a pleasing syntax highlighting

Comment (dckc) [reply packages/ERTP/docs/INPUT_VALIDATION.md:79]: some browsing around suggests that it is correct; that is: `console` is designed to distinguish input from output, which is what's going on here.

**Review by dckc — APPROVED** [review]
ok, I guess.

cc @warner since the main impact seems to be in swingset
